docs/v1/user-story

```diff
@@ -0,0 +1,123 @@
+I now get paid to play with this which is cool, but now I’m starting to get support tickets for it.
+
+Now I need to maintain multiple instances of this.  I have test and staging instances.  I also have two to three versions 
+under development at any given time.  I can deploy this thing with my eyes closed, but the interns have been accidentally 
+creating some difficulties for me.  They sometimes do strange configuration of test instances.  When I ask them to be be 
+more mindful about deployment, they push back on me.  They tell me that the company has adopted a standardized way to 
+templatize kubernetes hosted software that (a) handles parameterization so they know what they can play with and what I want 
+them to leave alone and (b) creates a standard provisioning API so that they can deploy my chat app the same way that they 
+deploy everything else in the company.  I check it out and it’s pretty cool - creating the template and parameters is 
+<simple process> and I can deploy the app with a single command.  The kubernetes UI even does the work of taking the 
+argument list and making fields for me in the deployment screen.  Even though it was not hard to deploy for me before, it’s 
+easier now even (whether from CLI or UI) and the interns are always deploying within the constraints I set.
+
+## 5. Attach a standard binding template
+## 6. Share service instances inside an org
+Someone in tech support just had the idea that they can use my chat app inside the call center system.  They need to connect 
+their app to my chat service the same way that I connected to the Rabbit service that IT maintains for me.  I call up the 
+Rabbit person and ask how they made that binding thing work.  Luckily, it was not hard at all.  I just did <simple workflow> 
+to make the service responsive to binding requests, deployed an instance and the customer support people were off to the 
+races.
+
+## 7. Buy access to a type from the marketplace  
+## 8. Bind a kubernetes service to a service outside kubernetes  	
+Those tech support people are expecting something a with a bit more enterprise capability.  They want the app to persist 
+information about chat logs and are are very concerned about the times we’ve lost it and brought it back empty.  This app 
+was just supposed to be for fun; I was not thinking about customer satisfaction audits.  If I could bring in a well managed 
+database without hiring a DBA, life would be great.  A friend of mine works at a database company that sells and manages 
+databases on the public cloud that we use.  She told me that if I was using Kubernetes or Cloud Foundry, this was trivial to 
+solve.  They implemented the exact same service broker model I had to implement.  I just needed to buy their database from 
+some store and with one API call they will deploy a managed database instance to the same cloud as my chat app and with a 
```

Comment:
the material thing is that this is kinda greybox.  It's managed like a blackbox, but the customer has some awareness of their own instance of the service like in whitebox.  Also, there is some shared control here as we see later in the story.  I think this type of scenario will be the bridge between downloading software and full on SaaS acceptance for the next few years.